recipe2023/index.css

```
@@ -0,0 +1,50 @@
+.container {
+  margin-left: auto;
+  margin-right: auto;
+  width: 1080px;
+  padding-top: 5%;
+}
+
+.centered-text {
+  text-align: center;
+}
+.main-title {
+  padding: 500px;
+}
+.highlight-title {
+  padding-bottom: 10%;
+}
+
+.column {
+  /* //splits half and half: allows us to create two rows in this case */
+  flex: 50%;
+}
+
+/* Clear floats after the columns */
+.row {
+  display: flex;
+}
+
+.removebullets {
+  margin-left: 100px;
+  margin-right: 100px;
+}
+.removebullets2 {
+  list-style: none;
+  margin-left: 100px;
+  margin-right: 100px;
+}
```

```suggestion
.removebullets {
  list-style: none;
}

.mx-100px {
  margin-left: 100px;
  margin-right: 100px;
}
```

A single HTML element can have any number of classes attached to it. So in cases where you're duplicating styles, you can create two classes and use them together. e.g.

```
<ul class="removebullets mx-100px">
  ...
</ul>
```